plugins/woocommerce/tests/e2e-pw/tests/shopper/cart.spec.js

```diff
@@ -73,18 +73,15 @@ test.describe( 'Cart page', () => {
 	test( 'should increase item quantity when "Add to cart" of the same product is clicked', async ( {
 		page,
 	} ) => {
-		await page.goto( '/shop/' );
-		await page.click(
-			`a[data-product_id='${ productId }'][href*=add-to-cart]`
-		);
-		// Once the view cart link is visible, item has been added
-		await page.waitForSelector( 'a.added_to_cart' );
```

Comment:
I had to remove this assertion because there had been cases where this test ran in parallel with `cart-redirection.spec.js`, which interferes with this test because the latter enables cart redirection, thus causing this assertion to fail.